src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/controller/ItemGroupController.java

```diff
@@ -86,8 +90,14 @@ public ResponseEntity<Object> createItemGroup(final @RequestHeader(X_REQUEST_ID_
             final ItemGroupData savedItemGroupData = itemGroupsService.createItemGroup(itemGroupData);
             return buildCreateSuccessResponse(xRequestId, savedItemGroupData);
         }
+        catch (IllegalArgumentException ex) {
+            return buildServerErrorResponse(xRequestId, ex, itemGroupData, BAD_REQUEST);
+        }
+        catch (MongoOperationException ex) {
+            return buildServerErrorResponse(xRequestId, ex, itemGroupData, INTERNAL_SERVER_ERROR);
+        }
         catch (Exception ex) {
-            return buildErrorResponse(xRequestId, ex, itemGroupData);
+            return buildServerErrorResponse(xRequestId, ex, itemGroupData, SERVICE_UNAVAILABLE);
```

Comment:
Looking at this again, I am not sure it makes a lot of sense to report `SERVICE_UNAVAILABLE` when we have an unknown error. I say that, because whatever the error is, it does not necessarily mean the service is down or not available. You might want to actually deal with this in the same way as the `MongoOperationException`, except you may of course already have logged more info about the former?